advanced/src/utils.ts

```diff
@@ -9,16 +9,14 @@ export interface Context {
 export function getUserId(ctx: Context) {
   const Authorization = ctx.request.get('Authorization')
   if (Authorization) {
-    const token = Authorization.replace('Bearer ', '')
-    const { userId } = jwt.verify(token, process.env.APP_SECRET) as { userId: string }
-    return userId
+    try {
```

Comment:
We had the same ideas how to improve that function :) As discussed in Slack, this is my take on it:

```
const Authorization = context.request.get("Authorization") || "";
const token = Authorization.replace("Bearer ", "");

const contents = jwt.verify(token, process.env.APP_SECRET);

if (!contents || !contents.userId) {
  throw new Error("Invalid token.");
}
```

Happy to hear your thoughts :)